src/Datasource/EntityTrait.php

```diff
@@ -375,6 +375,87 @@ public function has($property)
     }
 
     /**
+     * Returns whether this entity contains a property named $property
+     * that contains a non-null value.
+     *
+     * ### Example:
+     *
+     * ```
+     * $entity = new Entity(['id' => 1, 'name' => null]);
+     * $entity->has('id'); // true
+     * $entity->has('name'); // false
+     * $entity->has('last_name'); // false
+     * ```
+     *
+     * You can check multiple properties by passing an array:
+     *
+     * ```
+     * $entity->has(['name', 'last_name']);
+     * ```
+     *
+     * All properties must not be null to get a truthy result.
+     *
+     * When checking multiple properties. All properties must not be null
+     * in order for true to be returned.
+     *
+     * @param string|array $property The property or properties to check.
+     * @return bool
+     */
+    public function has($property)
```

Comment:
I think there should either be just has() or hasProperty(), not both of them as they do the same (aka deprecation may be required).

I do not care much which one should be in, but suspect that for newbies latter is better as it clearly tells what it does.